metaflow/extension_support/__init__.py

```diff
@@ -695,132 +847,29 @@ def _get_extension_packages(ignore_info_file=False, restrict_to_directories=None
-                            )
-                    # Check for meta files for this package
-                    meta_files = [
-                        x for x in map(EXT_META_REGEXP.match, files) if x is not None
-                    ]
-                    if meta_files:
-                        # We should have one meta file at most
-                        if len(meta_files) > 1:
-                            raise RuntimeError(
-                                "Package at '%s' defines more than one meta file: %s"
-                                % (
-                                    package_path,
-                                    ", and ".join(
-                                        ["'%s'" % x.group(0) for x in meta_files]
-                                    ),
-                                )
-                            )
-                        else:
-                            meta_module = ".".join(
-                                [relative_module, meta_files[0].group(0)[:-3]]
-                            )
+            state = {
+                "name": package_name,
+                "files": [],
+                "full_path_files": [],
+                "meta_module": None,
+            }
 
-                elif cur_depth > base_depth + 1:
-                    # We want at least a top-level name and something under
-                    tl_name = parts[base_depth]
-                    tl_fullname = "%s[%s]" % (package_path, tl_name)
-                    prefix_match = parts[base_depth + 1 :]
-                    for idx, ext_list in enumerate(list_ext_points):
-                        if prefix_match == ext_list:
-                            # We check to see if this is an actual extension point
-                            # or if we just have a directory on the way to another
-                            # extension point. To do this, we check to see if we have
-                            # any files or directories that are *not* directly another
-                            # extension point
-                            skip_extension = len(files) == 0
-                            if skip_extension:
-                                next_dir_idx = len(list_ext_points[idx])
-                                ok_subdirs = [
-                                    list_ext_points[j][next_dir_idx]
-                                    for j in range(0, idx)
-                                    if len(list_ext_points[j]) > next_dir_idx
-                                ]
-                                skip_extension = set(dirs).issubset(set(ok_subdirs))
-
-                            if skip_extension:
-                                _ext_debug(
-                                    "    Skipping '%s' as no files/directory of interest"
-                                    % _extension_points[idx]
-                                )
-                                continue
+            for root, _, files in os.walk(package_path):
```

Comment:
Consider whether we can set `followlinks` here. Bazel preps sandboxed directories using a lot of symlinks, so might get tripped up without this.

```suggestion
            for root, _, files in os.walk(package_path, followlinks=True):
```